environment.yaml

```diff
@@ -1,29 +1,16 @@
-name: 'collision_prediction'
-channels:
-  - conda-forge
-  - defaults
-dependencies:
-  - ipykernel
-  - matplotlib>=3.2.2
-  - scikit-learn>=1.0
-  - pandas>=1.3.*
-  - requests>=2.24.0
-  - altair 
-  - vega_datasets
-  - graphviz
-  - python-graphviz
-  - jinja2
-  - pip
-  - lightgbm
-  - py-xgboost==1.3.0
-  - catboost
-  - spacy
-  - wikipedia
-  - nltk
-  - eli5
-  - shap
-  - seaborn
-  - imbalanced-learn
-  - pip:
-    - mglearn
-    - psutil>=5.7.2
\ No newline at end of file
+name: collision_prediction
+channels:
+  - conda-forge
+  - defaults
+dependencies:
```

Comment:
Are you saying that altair saver didn't automatically include those?

Reply:
> > > > > > The column 'C_MNTH' contains both integers and strings so that we cannot apply OHE on it in the ML script, could you fix this?
> > > > > 
> > > > > 
> > > > > Should I convert them all to strings?
> > > > 
> > > > 
> > > > Yes please, since we have "missing" in the column which cannot be integer.
> > > 
> > > 
> > > Alright I think I've fixed this
> > 
> > 
> > Wait a minute I just realized that what I do isn't going to affect the training csv file. I think you need to add a line in your code that changes them all to strings (after you split them into X_train and y_train so that y_train remains boolean). You can do this with X_train.astype("string")
> 
> I tried X_train.astype("string") but the output type is still integer and string. Can't figure out why.

What error is it giving you?

Reply:
yeah altair saver wouldn't include thos
<img width="872" alt="Screen Shot 2021-11-27 at 12 19 05 PM" src="https://user-images.githubusercontent.com/94651544/143721128-bbdb0037-97a8-4ec3-bfc0-7fc79f0d5c8c.png">
e: